docs/adr/0003-definition-of-terms.md

```diff
@@ -0,0 +1,66 @@
+# 1. Record architecture decisions
```

Comment:
Minor one but the title should reflect the specific ADR e.g `0003 Definition of Terms`